lib/ord.js

```diff
@@ -105,6 +107,13 @@ const _getPackages = (policyLevel, appConfig) =>
             .packages(appConfig.appName, policyLevel, appConfig.ordNamespace)
             .slice(0, 1);
 
+const _getEntityTypes = (appConfig, packageIds) => {
```

Comment:
I have no better proposal and I'm open for any other proposal. 😊 The reason why I named the function this way is because the result of the function execution is `entityTypes` property/resource of the ORD document.

Reply:
Yeah, just leave it, I don't have better proposal as well.